__tests__/test-prepareCommit.js

```
@@ -0,0 +1,79 @@
+    const decoded = Buffer.from(payload.files[0].contents, 'base64').toString()
+    expect(decoded.length).toBeGreaterThan(0)
+  })
+
+  it('returns parentOid after a commit exists', async () => {
+    const { fs, dir } = await makeFixture('test-add')
+    await init({ fs, dir })
+    await add({ fs, dir, filepath: 'a.txt' })
+    const oid = await commit({
+      fs,
+      dir,
+      author: { name: 'Test', email: 'test@test.com', timestamp: 1000000000, timezoneOffset: 0 },
```

ESLint

[error] Replace `·name:·'Test',·email:·'test@test.com',·timestamp:·1000000000,·timezoneOffset:·0` with `⏎········name:·'Test',⏎········email:·'test@test.com',⏎········timestamp:·1000000000,⏎········timezoneOffset:·0,⏎·····`

(prettier/prettier)